File: clients/java/src/main/java/spb_ai_champ/model/DebugData.java
package spb_ai_champ.model;

import spb_ai_champ.util.StreamUtil;

/**
 * TODO - Document
 */
public class DebugData {

    public DebugData() {
    }

    /**
     * Read DebugData from input stream
     */
    public static DebugData readFrom(java.io.InputStream stream) throws java.io.IOException {
        return new DebugData();
    }

    /**
     * Write DebugData to output stream
     */
    public void writeTo(java.io.OutputStream stream) throws java.io.IOException {
    }

    /**
     * Get string representation of DebugData
     */
    @Override
    public String toString() {
        StringBuilder stringBuilder = new StringBuilder("DebugData { ");
        stringBuilder.append(" }");
        return stringBuilder.toString();
    }
}

File: clients/kotlin/src/main/kotlin/spb_ai_champ/model/DebugData.kt
package spb_ai_champ.model

import spb_ai_champ.util.StreamUtil

/**
 * TODO - Document
 */
class DebugData {

    constructor() {
    }

    /**
     * Write DebugData to output stream
     */
    @Throws(java.io.IOException::class)
    fun writeTo(stream: java.io.OutputStream) {
    }

    /**
     * Get string representation of DebugData
     */
    override fun toString(): String {
        var stringBuilder = StringBuilder("DebugData { ")
        stringBuilder.append(" }")
        return stringBuilder.toString()
    }

    companion object {
        /**
         * Read DebugData from input stream
         */
        @Throws(java.io.IOException::class)
        fun readFrom(stream: java.io.InputStream): DebugData {
            return DebugData()
        }
    }
}